ambry-store/src/test/java/com.github.ambry.store/IndexTest.java

```diff
@@ -1882,7 +1950,7 @@ private void findDeletedEntriesSinceIndexBasedTokenForOffsetInJournalTest() thro
     }
     maxSize += size;
 
-    StoreFindToken expectedEndToken = new StoreFindToken(nextIndexSegmentStartOffset, sessionId);
+    StoreFindToken expectedEndToken = new StoreFindToken(nextIndexSegmentStartOffset, sessionId, incarnationId, false);
     doFindDeletedEntriesSinceTest(startToken, maxSize, expectedKeys, expectedEndToken);
   }
 
```

Comment:
might have missed it, but do you have a test for changing incarnation ID in this file ? `PersistentIndexTest.java` is going to be deleted soon (you need not even add a test there).